.github/workflows/ci_scripts/ghcr_utils.py

```
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+"""
+Copyright (c) 2017-2021 ARM Limited. All rights reserved.
+SPDX-License-Identifier: Apache-2.0
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations 
+"""
+import click
+import requests
+import logging
+import sys
+import time
+import json
+import subprocess
+
+"""
+This file contains ghcr utlity wrapper used for:
+- retrieving digest of docker image
+- deleting images in ghcr
+"""
+@click.command()
+@click.pass_context
+@click.option("-r", "--repository", required=True)
+@click.option("-t", "--tag", required=True)
+@click.option("-p", "--platform", required=False)
+def get_digest(ctx, repository, tag, platform=None):
+    command = f"docker run quay.io/skopeo/stable --creds={ctx.obj['username']}:{ctx.obj['passwd']} inspect docker://ghcr.io/{ctx.obj['username']}/{repository}:{tag} --raw"
+    output = subprocess.run(command.split(), stdout=subprocess.PIPE).stdout.decode(
```

Do we need to handle errors here?

---

I will add check=True and let it crash. Any failure will make the CI fails, so errors are handled in away.